hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/ha/HASecurityUtils.java

```diff
@@ -262,7 +262,7 @@ private static PKCS10CertificationRequest generateCSR(
         client.getPrivateKey());
 
     // Get host name.
-    String hostname = scmAddress.getAddress().getHostName();
+    String hostname = scmAddress.getHostName();
```

Comment:
Is this change necessary?

Reply:
I cannot exactly remember. I think I got some cert issue when deploying to k8s. But when I just tested this, it seems both of these correctly return the hostname. I cannot remember now why I have this change. Are you OK to keep my change, or you think I should revert it? Please let me know. Thanks

Reply:
I'm ok with keeping it, I think it does the same thing.  I was just wondering.